### PR TITLE
Fix restart/interrupt kernel for interactive window script file

### PR DESCRIPTION
**Pull request by joyceerhl**
Fix https://github.com/microsoft/vscode-jupyter/issues/7158

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1185,19 +1185,19 @@
...
                     "command": "jupyter.restartkernel",
                     "title": "%jupyter.command.jupyter.restartkernel.title%",
                     "category": "Jupyter",
-                    "when": "jupyter.haveinteractive && jupyter.ispythonorinteractiveeactive && isWorkspaceTrusted"
+                    "when": "jupyter.ispythonorinteractiveeactive && isWorkspaceTrusted"
                 },
                 {
                     "command": "jupyter.notebookeditor.undocells",
                     "title": "%jupyter.command.jupyter.undocells.title%",
                     "category": "Notebook",
```
Comment (IanMatthewHuff) [inline package.json:1199]: You want "havenativeredoablecells" for undo?

**Review by IanMatthewHuff — CHANGES_REQUESTED** [review]
When condition might be off? Looking at the code I'm not even sure if that havenativeredoablecells is hooked up.

**Comment by IanMatthewHuff** [comment]
@joyceerhl BTW, I see that you had a Prettier failure. For some reason in my PR Prettier had failed to run for me locally, I had to restart VS Code before Prettier worked. Not sure why...

File: package.json (hunk 1 above)
Comment (joyceerhl) [reply package.json]: Oops, that was not intentional

**Review by IanMatthewHuff — APPROVED** [review]

**Pull request merged**